src/signals/incident-management/containers/IncidentOverviewPage/saga.js

```diff
@@ -9,7 +9,7 @@ import { requestIncidentsSuccess, requestIncidentsError, filterIncidentsChanged,
 import { makeSelectFilterParams } from './selectors';
 
 export function* fetchIncidents(action) {
-  const requestURL = `${CONFIGURATION.API_ROOT}signals/auth/signal/`;
+  const requestURL = `${CONFIGURATION.API_ROOT}signals/v1/private/signals/`;
```

Comment:
Shouldn't we be moving this to `shared/services/configuration`?

Reply:
niet nu voor deze pr